server/src/test/java/io/crate/integrationtests/AnalyzeITest.java

```diff
@@ -56,9 +56,10 @@ public void test_analyze_statement_refreshes_table_stats_and_stats_are_visible_i
         assertThat(((Float) row[0]).doubleValue()).isCloseTo(0.166, withPercentage(1));
         assertThat(row[1]).isEqualTo(DataTypes.INTEGER.fixedSize());
         assertThat(row[2]).isEqualTo(3.0f);
-        assertThat(((List<String>) row[3])).isEmpty();
-        assertThat(((List<Double>) row[4])).isEmpty();
-        assertThat(((List<String>) row[5])).containsExactly("1", "2", "3");
+        assertThat(((List<String>) row[3])).containsExactly("3", "2", "1");
```

Comment:
Doesn't matter I guess that the order is inverted?

Reply:
This was previously the histogram bounds, but we're getting the most common values back instead (which are always in frequency order).  I'm not sure why it wasn't returning the most common values before, to be honest - it's a small enough set that it should have always been returning them?  And histogram bounds don't contain any value that appears in MCV, so if you return all values as part of MCV then histogram bounds will be empty.